### PR TITLE
Create basic-implementation.md

### PR DESCRIPTION
**Pull request by ctmitchell28**

### CODE REVIEW

File: lightning-tag/basic-implementation.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+!> This document provides information on how to implement Lightning Tag in its basic form. Throughout this page, there will be links to documentation detailing additional features and use-cases that can be deployed
+
+Setup of the Lotame Lightning Tag javascript on your website involves an input object that gets passed to an initialization function that prepares a window namespace on your page for the javascript tag itself. 
+
+Lotame recommends loading the Lightning Tag directly on your page, in the <head> section. While not always possible, Lotame recommends against loading through a tag manager such as Google Tag Manager (GTM)
+
+The basic setup looks like the below:
+
+```javascript
```
Comment (singhald) [inline lightning-tag/basic-implementation.md:11]: does this need to include the js file itself in a script src tag?  in itself this doesn't need that code, but for this code to do anything it does?

Comment (singhald) [reply lightning-tag/basic-implementation.md:11]: or mention that this is just the configuration and you still need to load the actual code?

File: lightning-tag/basic-implementation.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+# Base Lightning Tag Implementation 
+
+!> This document provides information on how to implement Lightning Tag in its basic form. Throughout this page, there will be links to documentation detailing additional features and use-cases that can be deployed
+
+The base version of Lightning Tag will only:
+
+1. Execute data collection rules that you have configured in cooperation with your Lotame account representative
+1. Fire subscribed sync pixels
```
Comment (noahpc) [inline lightning-tag/basic-implementation.md]: and export beacons

File: lightning-tag/basic-implementation.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+This version of the tag is right for you if:
+
+1. 1st party data collection is the only desired use-case and you will not be passing data directly into the tag; instead relying on data collection rules configured within the DMP in cooperation with your Lotame account representative. 
+   * [How to pass data directly into the tag](lightning-tag/data-collection.md)
+1. You do not need audiences to be returned to the page for client-side activation (example passing audiences into your [Google Ad Manager](lightning-tag/implementation-google-ad-manager.md) calls). 
+   * [How to receive information on the profile](lightning-tag/detailed-reference?id=config-object.md)
+
+
+The base verison of Lightning Tag:
```
Comment (noahpc) [inline lightning-tag/basic-implementation.md]: typo - version

File: lightning-tag/basic-implementation.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
+# Base Lightning Tag Implementation 
+
+!> This document provides information on how to implement Lightning Tag in its basic form. Throughout this page, there will be links to documentation detailing additional features and use-cases that can be deployed
+
+The base version of Lightning Tag will only:
+
+1. Execute data collection rules that you have configured in cooperation with your Lotame account representative
+1. Fire subscribed sync pixels
+
+This version of the tag is right for you if:
```
Comment (noahpc) [inline lightning-tag/basic-implementation.md]: I think we can remove this section entirely and rely on the fact that is named Basic Implementation and there are specific links at the bottom for adding these functions. I would rather people start with this (incorrectly) and have to add to it than start with one of the more complicated (incorrectly) and be making a bunch of unnecessary work for turnstyle.

File: lightning-tag/basic-implementation.md (hunk 1 above)
Comment (noahpc) [reply lightning-tag/basic-implementation.md:11]: We should include the script tag as well as the header links for preconnect stuff.

**Review by singhald — APPROVED** [review]

File: lightning-tag/basic-implementation.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+```javascript 
+<head>
+  <link rel="preconnect" href="https://tags.crwdcntrl.net">
+  <link rel="preconnect" href="https://bcp.crwdcntrl.net">
+  <link rel="dns-prefetch" href="https://tags.crwdcntrl.net">            
+  <link rel="dns-prefetch" href="https://bcp.crwdcntrl.net">
+  
+<script>
+  ! function() {
+    var lotameClientId = '<lotameClientId>';
```
Comment (noahpc) [inline lightning-tag/basic-implementation.md]: Possibly add a note to indicate to replace <lotameClientId> with your client id in both places.

File: lightning-tag/basic-implementation.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
+# Basic Lightning Tag Implementation 
+
```
Comment (noahpc) [inline lightning-tag/basic-implementation.md:2]: @ErikMHummel  - where is this file going to live in the structure of the site?

File: lightning-tag/basic-implementation.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+  } ();
+</script>
+
+<script async src="https://tags.crwdcntrl.net/lt/c/<lotameClientId>/lt.min.js"></script>
+</head>
+```
+
+Note:
+
+* Setup of the Lotame Lightning Tag javascript on your website involves an input object that gets passed to an initialization function that prepares a window namespace on your page for the javascript tag itself.
```
Comment (noahpc) [inline lightning-tag/basic-implementation.md]: This first bullet point also seems superfluous.

**Review by noahpc — APPROVED** [review]

**Review by markaconrad — APPROVED** [review]

File: lightning-tag/basic-implementation.md (hunk 6 above)
Comment (ErikMHummel) [reply lightning-tag/basic-implementation.md:2]: We would need to update `_sidebar.md` to add it and then a snippet on `index.md` to call it out.

**Review by ErikMHummel — APPROVED** [review]

File: lightning-tag/basic-implementation.md (hunk 6 above)
Comment (noahpc) [reply lightning-tag/basic-implementation.md:2]: @ErikMHummel - can you work with Charlie to show him how to do that?

Comment (ErikMHummel) [reply lightning-tag/basic-implementation.md:2]: Yup, I have that on my list for today.

Comment (ErikMHummel) [reply lightning-tag/basic-implementation.md:2]: Done @noahpc, please review and if all looks good merge.

**Review by noahpc — APPROVED** [review]

**Pull request merged**